Tests/XCUITests/BaseTestCase.swift

```diff
@@ -274,4 +274,29 @@ extension XCUIElement {
             coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.5)).tap()
         }
     }
+
+    /// Tap at @offsetPoint point in @self element view
+    func tapAtPoint(_ offsetPoint: CGPoint) {
```

Comment:
We should add a comment that this works for the current simulator used iPhone 14 Plus. If tests run on a different simulator, tests fail